### DISCUSSION TITLE
Using an IF statement in Templater

### DISCUSSION THREAD

**Discussion by jacrook** [SilentVoid13/Templater#1213] [Help]
**Environment**
Obsidian 1.4.14 
Plugins: 
- Templater
- Omnivore Docs Plugin

**Steps to reproduce**
- Added the If statement below in the Omnivore Docs template in Obsidian 

**Expected Result**
- Goal state - If the passed original URL is a youtube like the IF will equal an embed of the video
- Else the standard content is passed

**Actual Result**
- I am 99.9 percent sure the I am not using the correct IF statement syntax
- Right now if it passes it would say Pizza or Elese it says taco
- Actual result passes everything as text

<img width="792" alt="CleanShot 2023-10-03 at 17 07 05@2x" src="https://github.com/SilentVoid13/Templater/assets/3718579/bf99dfaa-33e1-4079-9813-d12a266f553d">

```
<%*
let omniURL = '{{original_url}}';
if (!(tp.original_url.startsWith('https://youtube.com')))
{
   # pizza
<%* } else { %>

# Taco

<%* } %>
```

**Comment by Zachatoo** [comment]
As far as Templater goes, you were missing a closing `%>` 

```js
<%*
if (true) {
// don't forget the closing tag below!
-%>
# pizza

<%* } else { -%>
# Taco

<%* } %>
```

I would also be surprised if `tp.original_url` was available, but maybe the Omnivore Docs Plugin attaches stuff to `tp`? That would be kinda odd.

Also, remember that you have to run a Templater command from the command palette in order for the template to execute and replace it's contents.
